components/core/src/clp_s/ArchiveReader.hpp

```diff
@@ -54,18 +104,42 @@ class ArchiveReader {
      */
     void close();
 
+    /**
+     * @return The schema ids in the archive.
```

Comment:
Probably worth a comment that this schema_ids vector also defines the order that tables should be read in to avoid seeking backwards.